x-pack/plugins/encrypted_saved_objects/server/create_migration.ts

```diff
@@ -80,20 +92,40 @@ export const getCreateMigration = (
     const encryptDescriptor = { id, type, namespace: encryptedDoc.namespace };
 
     // decrypt the attributes using the input type definition
-    // then migrate the document
-    // then encrypt the attributes using the migration type definition
-    return mapAttributes(
-      migration(
-        mapAttributes(encryptedDoc, (inputAttributes) =>
-          inputService.decryptAttributesSync<any>(decryptDescriptor, inputAttributes, {
+    // if an error occurs during decryption, use the shouldMigrateIfDecryptionFails flag
+    // to determine whether to throw the error or continue the migration
+    // if we are continuing the migration, strip encrypted attributes from the document using stripOrDecryptAttributesSync
+    const documentToMigrate = mapAttributes(encryptedDoc, (inputAttributes) => {
+      try {
+        const decryptedAttributes = inputService.decryptAttributesSync<any>(
+          decryptDescriptor,
+          inputAttributes,
+          { convertToMultiNamespaceType }
+        );
+        return decryptedAttributes;
```

Comment:
__nit:__
```suggestion
        return inputService.decryptAttributesSync<any>(decryptDescriptor, inputAttributes, {
          convertToMultiNamespaceType,
        });
```

Reply:
Updated in https://github.com/elastic/kibana/pull/105968/commits/1365d05c60aeae8c4bd64962cfa6f93293dc4e0d